build-tools/packages/build-cli/src/commands/release/report-unreleased.ts

```diff
@@ -154,19 +154,39 @@ async function updateReportVersions(
 	version: string,
 	log: Logger,
 ): Promise<void> {
+	const clientPackage = "@fluidframework/aqueduct";
```

Comment:
Still not a fan of having a hardcoded package name here for a general check. Using `Object.keys(report)[1]` (and throwing if there are none) wasn't feasible?

Reply:
that's feasible. The release report is typically sorted in ascending order, but just thinking, if there's an error with the sorting and the server package ends up at Object.keys(report)[1], it would update the versions incorrectly.

Reply:
That's a fair concern, I was thinking the report would only have the client packages but I'm not sure exactly what it contains. In that case maybe using the `fluid-framework` package as the target, since that one is very unlikely to ever go away or change names, might be ok?